src/connection.js

```diff
@@ -77,31 +91,19 @@ class Connection extends EventEmitter {
         textsize: DEFAULT_TEXTSIZE,
         trustServerCertificate: true,
         useColumnNames: false,
-        useUTC: true
+        useUTC: true,
+        language: DEFAULT_LANGUAGE,
+        dateFormat: DEFAULT_DATEFORMAT
       }
     };
 
+    self.config = _.merge(defaults, config);
```

Comment:
Can you change this back to not use `_.merge`? The idea here is to make sure that the connection configuration will never end up with additional config properties other than what is supported.

We had similar code as above before, but that lead us to not have a single, definitive list of supported options. See the discussion in https://github.com/tediousjs/tedious/pull/445.